docs/docs/cloud/concepts/api.md

```diff
@@ -12,6 +12,10 @@ An assistant is a configured instance of a [`CompiledGraph`][compiledgraph]. It
 
 The LangGraph Cloud API provides several endpoints for creating and managing assistants. See the <a href="../reference/api/api_ref.html#tag/assistantscreate" target="_blank">API reference</a> for more details.
 
+#### Configuring Assistants
+
+One of the benefits of LangGraph API is that it lets you create agents with different configurations. This is useful when you want to allow users to use different models or prompts when using your graph. This also allows you the ability to quickly test out different configurations without having to rewrite your graph every time. See <a href="https://langchain-ai.github.io/langgraph/cloud/how-tos/cloud_examples/configuration_cloud/">this</a> how-to for information on how to configure a deployed graph. 
```

Comment:
style preference (that I often deviate from): be very direct.

Instaed of "One of the benefits of LangGraph API is that it lets you create agents with different configurations. This is useful when you want to allow users to use different models or prompts when using your graph."

"You can save custom *assistants* from the same graph to set different default prompts, models, and other *configurations* without changing a line of code in your graph."

Reply:
Yup, this sounds much better - thanks for the feedback!